packages/web/src/features/languages/ManageLanguageView.tsx

```diff
@@ -74,6 +74,7 @@ function useUpdateLanguageMemberMutation() {
       queryClient.invalidateQueries({
         queryKey: languageMembersQueryKey(code).queryKey,
       });
+      queryClient.invalidateQueries({ queryKey: ['session'] });
```

Comment:
In all of these places, it would be good to confirm the affected user is the current user before we force the session to reload. That prevents an unnecessary network request

Reply:
Ah, makes sense.